### PR TITLE
change organization references from projectatomic to containerbuildsy…

### PR DESCRIPTION
**Pull request by rcerven**
…stem

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: docs/osbs_instance_setup.md [hunk 1]
```diff
@@ -5,12 +5,12 @@ OSBS consists of several components, namely:
...
  * [Docker](https://www.docker.com/)
  * [OpenShift 3](https://www.openshift.org/) - Enterprise and Origin both work
- * [atomic-reactor](https://github.com/projectatomic/atomic-reactor)
+ * [atomic-reactor](https://github.com/containerbuildsystem/atomic-reactor)
  * (optional) authenticating proxy based on Apache httpd
  * (optional) docker registry where built images are going to be pushed
 
 The recommended way to install OSBS is to use our
-[ansible playbook](https://github.com/projectatomic/ansible-osbs).
+[ansible playbook](https://github.com/containerbuildsystem/ansible-osbs).
```
Comment (athos-ribeiro) [inline docs/osbs_instance_setup.md]: This repo was not migrated (404)

File: docs/osbs_instance_setup.md [hunk 2]
```diff
@@ -316,11 +316,11 @@ $ oadm policy add-role-to-group edit system:authenticated
 
 In order to build images, you need to have a build image. It is the image used
 by OpenShift to perform builds. The image has installed component called
-[atomic-reactor](https://github.com/projectatomic/atomic-reactor), which
+[atomic-reactor](https://github.com/containerbuildsystem/atomic-reactor), which
 performs the build itself.
 
 Please see the project's documentation for more complete information. There's
-also [ansible role](https://github.com/projectatomic/ansible-role-atomic-reactor)
+also [ansible role](https://github.com/containerbuildsystem/ansible-role-atomic-reactor)
```
Comment (athos-ribeiro) [inline docs/osbs_instance_setup.md]: This repo was not migrated (404)

File: osbs/utils.py [hunk 3]
```diff
@@ -567,7 +567,7 @@ class Labels(object):
     Provide access to a set of labels which have specific semantics
 
     The set of labels comes from here:
-    https://github.com/projectatomic/ContainerApplicationGenericLabels
+    https://github.com/containerbuildsystem/ContainerApplicationGenericLabels
```
Comment (athos-ribeiro) [inline osbs/utils.py]: This repo was not migrated (404)

File: docs/osbs_instance_setup.md (hunk 1 above)
Comment (rcerven) [reply docs/osbs_instance_setup.md]: fixed

File: docs/osbs_instance_setup.md (hunk 2 above)
Comment (rcerven) [reply docs/osbs_instance_setup.md]: fixed

File: osbs/utils.py (hunk 3 above)
Comment (rcerven) [reply osbs/utils.py]: fixed

**Pull request merged**